src/plugins/comps/components/CompsView.vue

```diff
@@ -0,0 +1,296 @@
+            "
+          >
+            {{ currentCompOutput }}
+          </template>
+          <template v-else> --- </template>
+        </span>
+      </div>
+    </section>
+    <section
+      id="telemetryReferenceSection"
+      class="c-comps__section c-comps__refs-and-controls"
+      aria-label="Derived Telemetry References"
+    >
+      <div class="c-cs__header c-section__header">
+        <div class="c-cs__header-label c-section__label">Telemetry References</div>
+      </div>
+
+      <div :class="['c-comps__refs-controls c-cdef__controls', { disabled: !parameters?.length }]">
+        <label v-if="isEditing" class="c-toggle-switch">
+          <input type="checkbox" :checked="testDataApplied" @change="toggleTestData" />
+          <span class="c-toggle-switch__slider" aria-label="Apply Test Data"></span>
+          <span class="c-toggle-switch__label">Apply Test Values</span>
+        </label>
+      </div>
+      <div class="c-comps__refs">
+        <div v-for="parameter in parameters" :key="parameter.keyString" class="c-comps__ref">
+          <span class="c-test-datum__string">Reference</span>
+          <input
+            v-if="isEditing"
+            v-model="parameter.name"
+            :aria-label="`Reference Name Input for ${parameter.name}`"
+            type="text"
+            class="c-input--md"
+            @change="updateParameters"
+          />
+          <div v-else class="--em">{{ parameter.name }}</div>
+          <span class="c-test-datum__string">=</span>
+          <span
+            class="c-comps__path-and-field"
+            :aria-label="`Reference ${parameter.name} Object Path`"
+          >
+            <ObjectPathString
+              :domain-object="compsManager.getTelemetryObjectForParameter(parameter.keyString)"
+              :show-object-itself="true"
+              class="--em"
+            />
+            <!-- drop down to select value from telemetry -->
+            <select v-if="isEditing" v-model="parameter.valueToUse" @change="updateParameters">
+              <option
+                v-for="parameterValueOption in compsManager.getMetaDataValuesForParameter(
+                  parameter.keyString
+                )"
+                :key="parameterValueOption.key"
+                :value="parameterValueOption.key"
+              >
+                {{ parameterValueOption.name }}
+              </option>
+            </select>
+            <div v-else>{{ parameter.valueToUse }}</div>
+          </span>
+
+          <span v-if="isEditing" class="c-test-datum__string">Test value</span>
+          <input
+            v-if="isEditing"
+            v-model="parameter.testValue"
+            :aria-label="`Reference Test Value for ${parameter.name}`"
+            type="text"
+            class="c-input--md"
+            @change="updateParameters"
+          />
+        </div>
+      </div>
+    </section>
+    <section
+      id="expressionSection"
+      aria-label="Derived Telemetry Expression"
```

Comment:
same here

Reply:
we're not using this in the test, so i can just remove it